website/integrations/services/gitea/index.md

```diff
@@ -69,3 +69,55 @@ Change the following fields
 ![](./gitea1.png)
 
 `Add Authentication Source` and you should be done. Your Gitea login page should now have a `Sign in With` followed by the authentik logo which you can click on to sign-in to Gitea with Authentik creds.
+
+## Helm Chart Configuration
+
+Authentik can be configured automatically in Gitea Kubernetes deployments via it's [Helm Chart](https://gitea.com/gitea/helm-chart/).
```

Comment:
Nit: should be lowercased as stated here https://goauthentik.io/developer-docs/docs/writing-documentation#general-guidelines